src/social_db/data/set/function_args/mod.rs

```diff
@@ -0,0 +1,39 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod with_json;
+mod with_json_file;
+mod with_text;
+mod with_text_file;
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = super::SetContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How do you want to pass the function call arguments?
+pub enum FunctionArgsType {
```

Comment:
These are not "function call arguments" from the user perspective. Please, come up with better naming for the module, struct, and documentation.

Reply:
Done